### PR TITLE
eliminate quotas warnings in vmwarevirtustream 2019-08-09-preview

### PR DESCRIPTION
**Pull request by ctaggart**
This eliminates some warning from https://github.com/Azure/azure-rest-api-specs/pull/9244 that was merged earlier today:

```
WARNING (PostOperationIdContainsUrlVerb/R2066/SDKViolation): OperationId should contain the verb: 'checkquotaavailability' in:'Quota_Get'. Consider updating the operationId

WARNING (EnumInsteadOfBoolean/R3018/ARMViolation): Booleans are not descriptive and make them hard to use. Consider using string enums with allowed set of values defined. Property: quotaEnabled
```

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json [hunk 1]
```diff
@@ -128,13 +128,13 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.VMwareVirtustream/locations/{location}/checkQuotaAvailability": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VMwareVirtustream/locations/{location}/quotas/checkAvailability": {
```
Comment (pilor) [inline specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json:131]: >/quotas/checkAvailability [](start = 95, length = 25)

this isn't a valid POST action URI.  Keep the URI the same, just update the operation ID to "Quotas_CheckQuotaAvailability" or something similar

**Review by pilor — CHANGES_REQUESTED** [review]
:clock1:

File: specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json (hunk 1 above)
Comment (ctaggart) [reply specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json]: I change it back, but the problem now is:
```
WARNING (OperationIdNounVerb/R1001/SDKViolation): Per the Noun_Verb convention for Operation Ids, the noun 'Quotas' should not appear after the underscore. Note: If you have already shipped an SDK on top of this spec, fixing this warning may introduce a breaking change.
    - file:///azure-rest-api-specs/specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json:137:8 ($.paths["/subscriptions/{subscriptionId}/providers/Microsoft.VMwareVirtustream/locations/{location}/checkQuotaAvailability"].post.operationId)
```

Comment (ctaggart) [reply specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json]: I see, it should just be `CheckQuotaAvailability`.

Comment (pilor) [reply specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json]: The operationId is something that is SDK specific so I'm signing off from the ARM side.  @allenjzhang what is the intended operation ID here for a provider level action based on the various warnings?

**Review by pilor — APPROVED** [review]

**Comment by ctaggart** [comment]
Can this be merged, please?

**Pull request merged**